emsdk.py

```diff
@@ -2118,6 +2102,12 @@ def update_emsdk():
     sys.exit(1)
   if not download_and_unzip(emsdk_zip_download_url, emsdk_path(), download_even_if_exists=True, clobber=False):
     sys.exit(1)
+
+  # attempt to run fetch_emscripten_tags if git is available.
+  git = GIT(must_succeed=False)
+  if not git:
+    print('Update complete, however skipped fetching the Emscripten tags, since git was not found, which is necessary for update-tags.', file=sys.stderr)
+    return
```

Comment:
instead of depending on this being the end of the function, how about `if git, update tags, else warn on no git`?